pkg/account/delete/loader.go

```diff
@@ -64,19 +64,15 @@ func LoadResourcesToDelete(fs afero.Fs, deleteFile string) (Resources, error) {
 }
 
 func readDeleteFile(context *loaderContext) (FileDefinition, error) {
-	targetFile, err := filepath.Abs(context.deleteFile)
```

Comment:
this just causes problems in test context with e.g. afero.
I've deleted it, as i think (?) it is not really necessary, please confirm